tests/altair_theme_test.py

```diff
@@ -0,0 +1,276 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+import jinja2
+import polars as pl
+
+import altair as alt
+from vega_datasets import data
+
+common_data = pl.DataFrame(
+    [
+        {"Index": 1, "Value": 28, "Position": 1, "Category": "A"},
+        {"Index": 2, "Value": 55, "Position": 2, "Category": "A"},
+        {"Index": 3, "Value": 43, "Position": 3, "Category": "A"},
+        {"Index": 4, "Value": 91, "Position": 4, "Category": "A"},
+        {"Index": 5, "Value": 81, "Position": 5, "Category": "A"},
+        {"Index": 6, "Value": 53, "Position": 6, "Category": "A"},
+        {"Index": 7, "Value": 19, "Position": 1, "Category": "B"},
+        {"Index": 8, "Value": 87, "Position": 2, "Category": "B"},
+        {"Index": 9, "Value": 52, "Position": 3, "Category": "B"},
+        {"Index": 10, "Value": 48, "Position": 4, "Category": "B"},
+        {"Index": 11, "Value": 24, "Position": 5, "Category": "B"},
+        {"Index": 12, "Value": 49, "Position": 6, "Category": "B"},
+        {"Index": 13, "Value": 87, "Position": 1, "Category": "C"},
+        {"Index": 14, "Value": 66, "Position": 2, "Category": "C"},
+        {"Index": 15, "Value": 17, "Position": 3, "Category": "C"},
+        {"Index": 16, "Value": 27, "Position": 4, "Category": "C"},
+        {"Index": 17, "Value": 68, "Position": 5, "Category": "C"},
+        {"Index": 18, "Value": 16, "Position": 6, "Category": "C"},
+    ]
+)
+
+
+bar = (
+    alt.Chart(common_data, title="Bar", width=480, height=150)
+    .mark_bar()
+    .encode(x=alt.X("Index:O").axis(offset=1), y=alt.Y("Value:Q"), tooltip="Value:Q")
+)
+
+line = (
+    alt.Chart(common_data, width=240, height=150, title="Line")
```

Comment:
### TODO
- Move all parts of the final chart definition into a single function
- Handle `textwrap.dedent`-ing -> reformat w/`ruff`
- Then make the source available